content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -154,7 +154,7 @@ configure cross-network gateways without using [mesh networks](/docs/reference/c
   ([Issue #25933](https://github.com/istio/istio/issues/25933))
 
 - **Added** support for Kubernetes versions 1.16 to 1.19 to Istio 1.8.
-  ([Issue #25793](https://github.com/istio/istio/issues/25793))
+  ([Pull Request #28814](https://github.com/istio/istio/pull/28814))
```

Comment:
This change seems gratuitous. Let's not make changes that require cherrypick unless they are bugs or broken links.

Reply:
I also thought we usually didn't point to PRs, although I've done it in the past.

Reply:
I have changed the URL to point to issues that automatically redirect to PR if the issue does not exist.

I think this change is required because https://github.com/istio/istio/issues/25793 displays the old supported version.